docs/releasing/publishing-a-pre-release.md

```diff
@@ -18,11 +18,11 @@ Projects can point to this branch in their package.json, instead of to the publi
 
 ## Publish a pre-release
 
-1. Run `git checkout -b BRANCH-NAME` to switch to the branch you want to pre-release.
+1. Check out the Git branch you want to pre-release.
```

Comment:
Would it be worth giving examples? For example:

`git checkout -b BRANCH-NAME` to checkout a new branch, or `git checkout BRANCH-NAME` to checkout an existing branch?

Reply:
Hmmm, we could but we need an optional extra step to make the code changes. Something like:

> 1. Run `git checkout -b BRANCH-NAME` to check out a new branch, or `git checkout BRANCH-NAME` to switch to the branch you want to pre-release.
> 2. Make the required changes and commit them.

Does this work @EoinShaughnessy?

Reply:
@hannalaakso Yeah, looks good 👍🏻

Reply:
@vanitabarrett I've made the required changes if you have time to take another look.